boards/arc/nsim_em/nsim_sem.yaml

```diff
@@ -0,0 +1,10 @@
+identifier: nsim_em
+name: EM Nsim simulator
+type: mcu
```

Comment:
type should be 'simulation' IMO, then we can use it in sanitycheck like qemu.

Reply:
To get this in sanitycheck is the main goal, indeed. Some help with that would be appreciated to get all the python script i/o redirection, flushing etc. correct.

Reply:
yes, will look into that